cypress/integration/tool_tests/swot-analysis_I.js

```diff
@@ -0,0 +1,250 @@
+        //Dann werden diese mit hilfe von FillFactors mit Testdaten aufgefüllt
+        cy.get("button[class='accordion-button collapsed']")
+          .each(($press) =>
+          {
+            var counter = -2; //Input des SWOT-Namens und Beshreibung werden nicht gezählt
+            if ($press.is(":visible"))
+            {
+              cy.wrap($press)
+              .click()
+            }
+
+            cy.wait(200)
+
+
+            cy.get("input")
+              .each(($countInput) =>
+              {
+                if ($countInput.is(":visible"))
+                {
+                  counter++
+                }
+              })
+
+            cy.get('div[class="addCard card"]')
+              .each(($add) =>
+              {
+                if ($add.is(":visible"))
+                {
+                  if(counter < 4)
+                  {
+                    for (let i = 2; i < 4; i++)
+                    {
+                      cy.wrap($add)
+                      .click()
+                      counter++
+
+                    }
+                  }
+                }
+              })
+
+            FillFactors();
+
+          })
+
+          cy.log("Additional Cards added and filled")
+
+          //
+          cy.contains("Speichern")
+          .click();
+          
+          //TODO: This POST gives back a NULL is that supposed to be so?
+          //cy.intercept('POST', /.*api\/saves.*/).as('overWrite')
+          /*
+          cy.get("@overWrite")
+          .its("response")
+          .should('include',
+          {
+            statusCode: null
+          })
+          */
+          
+      })
+
+})
+
+function FillFactors()
+{
+  function GetFillFunction (key, swot)
+  {
+   return ($input) => {
+      if($input.is(":visible"))
+        {
+          var regex = /([a-zA-Z]+)\[([0-9]+)]\[([a-zA-Z]+)]/;
+          var sampleText = $input.attr("name");
+
+
+          if(sampleText != null && sampleText.length > 0)
+          {
+            let result = sampleText.match(regex);
+
+            cy.wrap($input)
+            .clear()
+            .type(swot[result[1]][result[2]][key]);
+
+            CheckFactor(key,$input,swot,result);
+          }
+
+        }
+      }
+  }
+  //fixtures tootestdata swot werden hier genutzt
+  cy.fixture("tooltestdata").then(function (testdata)
+  {
+  this.testdata = testdata;
+  var swot = this.testdata.swot
+  cy.get("input")
+    .each(GetFillFunction("title", swot))
+
+  cy.get("textArea")
+    .each(GetFillFunction("desc", swot))
+  })
+
+  //Hier wird nur übeprüft ob die richtigen Daten im richtigen Feld landen.
+  function CheckFactor(key, $input, swot, result,)
+  {
+    if(key == "title")
+    {
+
+    cy.wrap($input).should('have.value', swot[result[1]][result[2]]["title"])
+
+    }else if (key == "desc")
+    {
+
+      cy.wrap($input).should('have.value', swot[result[1]][result[2]]["desc"])
+      
+    }
+    
+  }
+}
+function CreateSave()
```

Comment:
Move to command with arguments (name, data, tool_id, owner and description as command)